pyspecProcScripts/integral_w_error.py

```diff
@@ -13,6 +14,7 @@ def integral_w_errors(
     convolve_method="Gaussian",
     indirect="vd",
     direct="t2",
+    use_mask=False,
```

Comment:
this should not have `use_mask` defined.  Also, it should not have `error_path`, but rather `excluded_pathways`